### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by wmfwarren**
I redid it again with no innerHTML +=. Happier now.

The border does get biggified, but it is the same color as the background so you really need to look for it.

I was a little confused by a few tech reqs, but I think I got most of it.

### CODE REVIEW

File: js/carDescriptionChanger.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+var CarLot = (function(carLot){
+//gets and activates the selected card
+	carLot.selectCard = function (event, color){
+		targetCard = event.currentTarget;
+		//set target card styles
+		targetCard.style.borderWidth = "5px";
+		targetCard.style.backgroundColor = `${color}`;
+		targetCard.style.color = "white";
+		targetCard.classList.add("selected");
```
Comment (JoeShep) [inline js/carDescriptionChanger.js:6]: Other than the color, which comes from the data, the other attributes are better off being set in the CSS.
You can use the "selected" class to your advantage to set the different CSS when a card is clicked.

The background should be a different color form the border.

File: js/carLot.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    carLot.getInventory = function () { //this function gets and returns the inventory
+    	return inventory;
+    };
+  //This function loads all the information from the JSON file
+    carLot.loadInventory = function (popPage) { //this function loads and parses the JSON
+      var inventoryLoader = new XMLHttpRequest();
+
+      inventoryLoader.addEventListener("load", function () {
+      	inventory = JSON.parse(inventoryLoader.responseText);
```
Comment (JoeShep) [inline js/carLot.js]: indentation is inconsistent here. Seems to be only one space

File: js/carLot.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      	popPage(inventory);
+      });
+
+      inventoryLoader.open("GET", "inventory.json");
+			inventoryLoader.send();	
+    };
```
Comment (JoeShep) [inline js/carLot.js]: inconsistent indentation

File: js/carLotEvents.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+//This function activates the events for the DOM
+	carLot.activateEvents = function (){
+		var button = document.getElementById("textInputButton");
+		var inventory = CarLot.getInventory;
+		button.addEventListener("click", applyNewDescritionToCard);
+
+		for (let i = 0; i < CarLot.getInventory().cars.length; i++){
+			document.getElementById(`carCard--${i}`).addEventListener("click", function(click){
+				CarLot.selectCard(click, CarLot.getInventory().cars[i].color);
+			});
```
Comment (JoeShep) [inline js/carLotEvents.js:34]: You also need to add functionality to focus the input field when a card is clicked

**Comment by JoeShep** [comment]
This is looking good! Couple of items noted in the code. Also, you need to revisit the adding text to the cards when typing, This should happen as the user types, and you need to clear the input when a different card is selected.

Let me know when you've pushed up your tweaks and I'll take another look

**Comment by wmfwarren** [comment]
What is the button for if we are showing the text as they type? I was confused because I thought we had to use the button I think.

**Comment by wmfwarren** [comment]
I think it should be good now. The indentation is odd because some of it looks very different for me locally, but I think I got it fixed :)

**Comment by JoeShep** [comment]
👍

**Pull request merged**